.github/workflows/build.yml

```diff
@@ -53,7 +53,7 @@ jobs:
           java-version: 14
 
       - name: Upload Artifacts
-        run: ./gradlew uploadArchives
+        run: ./gradlew publish --no-daemon --no-parallel
```

Comment:
The daemon is already disabled via `GRADLE_OPTS` and we do not enable parallel in `gradle.properties` so neither should have an effect.

Reply:
Modified